include/bout/sys/expressionparser.hxx

```diff
@@ -41,6 +41,8 @@ class ParseException;
 #include <memory>
 #include <exception>
 
+using FieldGenerator_ptr = std::shared_ptr<FieldGenerator>;
```

Comment:
Is this name consistent with our conventions (I think it might not be, but not sure)?